ui/packages/consul-ui/app/components/consul/discovery-chain/utils.js

```diff
@@ -43,6 +43,7 @@ export const getSplitters = function(nodes) {
     const temp = item.Name.split('.');
     temp.reverse();
     temp.shift();
+    temp.shift();
```

Comment:
This line was the "bugfix" previous to this splitters were showing `service-name.nspace` instead of just `service-name`. I think theoretically splitters can actually be in a different namespace, so longer term we should display some sort of namespace here, but desired feature at the minute is to just show the `service-name`.

Reply:
So it turns out this wasn't a bugfix, it was an unfinished feature, and I added the feature to our mocks without realising it was not yet done in the backend. So until #11757 went in, this was essentially _introducing_ a bug.